client/my-sites/site-settings/site-admin-interface/index.js

```diff
@@ -135,57 +136,78 @@ const SiteAdminInterface = ( { siteId, isHosting } ) => {
 						<CardHeading id="admin-interface-style" size={ 20 }>
 							{ translate( 'Admin interface style' ) }
 						</CardHeading>
-						<p>
-							{ translate(
-								'Set the admin interface style for all users. {{supportLink}}Learn more{{/supportLink}}.',
-								{
+						{ isEnabled( 'layout/dotcom-nav-redesign-v2' ) ? (
+							<p className="form-setting-explanation">
+								{ translate( 'This setting has now moved to {{a}}Settings → General{{/a}}.', {
 									components: {
-										supportLink: (
-											<InlineSupportLink
-												supportContext="admin-interface-style"
-												showIcon={ false }
+										a: (
+											<a
+												href={ `/settings/general/${ siteSlug }#admin-interface-style` }
+												rel="noreferrer"
 											/>
 										),
 									},
-								}
-							) }
-						</p>
+								} ) }
+							</p>
+						) : (
+							<>
+								<p>
+									{ translate(
+										'Set the admin interface style for all users. {{supportLink}}Learn more{{/supportLink}}.',
+										{
+											components: {
+												supportLink: (
+													<InlineSupportLink
+														supportContext="admin-interface-style"
+														showIcon={ false }
+													/>
+												),
+											},
+										}
+									) }
+								</p>
+							</>
```

Comment:
Nitpick (non-blocking): I think the wrapping fragment is unnecessary:


```suggestion
							<p>
								{ translate(
									'Set the admin interface style for all users. {{supportLink}}Learn more{{/supportLink}}.',
									{
										components: {
											supportLink: (
												<InlineSupportLink
													supportContext="admin-interface-style"
													showIcon={ false }
												/>
											),
										},
									}
								) }
							</p>
						</>
```

Reply:
Done!